### ISSUE TITLE
[EI61][Analytics] Stats of endpoints which are reffering through message processors are not going to publish

### ISSUE THREAD

**Issue by Dilshani** [wso2/product-ei#366]
**Environment**
Operating Systems : Redhat and Debian
Java Version : JDK1.8
Packs : EI610 PreAlpha Pack
EI Setup: Cluster (3 nodes)
Analytics Setup: Cluster (2 nodes)
DB - Oracle 12c

**Preconditions**
Enabled publishing statistics in global level

**Steps to reproduce**
1. Add message store, scheduled forwarding message processor with an endpoint
2. Enable statistics in endpoint
3. Invoke proxy service which store message in message store
(Refer sample configuration)

**Expected Result**
Endpoint statistics should be displayed in dashboard

**Actual Result**
Endpoint statistics are not going to display in analytics dashboard.

Sample Configurations:

```
<endpoint name="TestEndpoint">
        <address statistics="enable" uri="http://www.mocky.io/v2/58d4e7c31000000501d0b0a2"/>
 </endpoint>
 <proxy name="testProxy" startOnLoad="true" transports="http https">
        <description/>
        <target>
            <inSequence>
                <log>
                    <property name="Message" value="=========This is testing=========="/>
                </log>
                <store messageStore="testMS"/>
            </inSequence>
        </target>
    </proxy>
 <messageStore name="testMS"/>
 <messageProcessor
       class="org.apache.synapse.message.processor.impl.forwarder.ScheduledMessageForwardingProcessor"
        messageStore="testMS" name="TestProcessor" targetEndpoint="testEndpoint">
        <parameter name="client.retry.interval">1000</parameter>
        <parameter name="throttle">false</parameter>
        <parameter name="max.delivery.attempts">4</parameter>
        <parameter name="member.count">1</parameter>
        <parameter name="max.delivery.drop">Disabled</parameter>
        <parameter name="interval">1000</parameter>
        <parameter name="is.active">true</parameter>
        <parameter name="target.endpoint">testEndpoint</parameter>
    </messageProcessor>
```

**Comment by nandika** [comment]
Code is not there in esb publisher to publish message processor endpoint statistics.

**Comment by AnuGayan** [comment]
I have tested this with EI analytics 6.4.0 and It is already fixed in EI 6.4.0
